### ISSUE TITLE
Separate DB connections for READ-ONLY vs WRITE queries

### ISSUE THREAD

**Issue by ghost** [Phorum/Core#502]
I put this as a 5.3 milestone with low priority, but it should be considered rather seriously.

In order to facilitate much larger forums spanning multiple servers it would be helpful to separate the DB connections such that.
DB reads would happen from a possibly different database server/connection than the writes.

it makes sense to architect a database with read-only slave replica's on the same host as the web server, and updates happening on the master DB which may be on a different host.
Once this code change is done, phorum is ready for a large scale multi-server load-balanced environment.

Reported by: freedman@freeformit.com
Imported from TRAC: http://trac.phorum.org/ticket/601
